CMakeModules/FindCBLAS.cmake

```diff
@@ -46,7 +46,7 @@ IF(PC_CBLAS_FOUND)
   SET(CBLAS_INCLUDE_DIR ${CBLAS_INCLUDE_DIRS})
 
   FIND_PACKAGE_HANDLE_STANDARD_ARGS(CBLAS DEFAULT_MSG CBLAS_LIBRARIES CBLAS_INCLUDE_DIR)
-  MARK_AS_ADVANCED(CBLAS_LIBRARIES CBLAS_INCLUDE_DIR)
+  MARK_AS_ADVANCED(CBLAS_LIBRARIES CBLAS_INCLUDE_DIR CBLAS_INCLUDE_DIRS)
```

Comment:
use lower case functions. One variable per line.

Reply:
That file is quite old, I was just sticking to whatever style that was present in the file. I can't change the function to lower case in just one line, it would look odd.

This file may need a rewrite and I don't want to mix those changes with this PR.